train.py

```diff
@@ -119,6 +122,7 @@ def train_model(args, device, parallel):
                 batch_size=int(args[ds_cfg]['batch_size']), shuffle=False, num_workers=int(args[ds_cfg]['num_workers']))
             print('Training model on the {} dataset'.format(args[ds_cfg]['ds_name']))
             train_val_loaders['valid'].append(valid_loader)
+    misc_utils.save_file(os.path.join(args['save_dir'], 'config.json'), args) # save config with actual mean and std used
```

Comment:
Moved config.json saving from main() to train_model() as the values actually used in training will be saved.